<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAnimalDialog</class>
 <widget class="QWidget" name="EditAnimalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование популяции</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>571</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Что вы хотите сделать?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="editActMonthButton">
        <property name="text">
         <string>Изменить месяцы активности</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editRepMonthButton">
        <property name="text">
         <string>Изменить месяцы размножения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editHumanMonthButton">
        <property name="text">
         <string>Изменить месяцы влияния человека</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="deleteAnimalButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>185</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
